en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -361,13 +362,19 @@ To create one or more new variables using the *Visual Editor* window:
 . In the *Visual Editor*, in the *Label* field, enter a unique name for the variable.
 +
 The name entered is unique only to this step. You can duplicate values in the *Label* field across different Variable steps. 
-. In the *Data Type* dropdown, select the data type you want to use for the variable. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, `Boolean`, `Date`, `DateTime`.
-. Optionally, to add additional variables in the *Visual Editor*, click *Add Variable*. You can create multiple variables in each Variable step. 
-. Click *Apply*. +
-The *Create Variables* window appears, displaying one or more new variables added to the flow. 
-. Optionally, to change the variables, click *Manage Variables* to open the Visual Editor or *Edit Schema* to open the JSON Schema. 
-. Specify the initial value for each variable. The default value is defined under each variable. If the field is empty, the value is set to the default value for that data type. You can manually enter a value, select any existing data pill (including variables) from your flow, or select the xref:ms_composer_custom_expression_editor.adoc#add-custom-expression-to-an-action[*Custom Expression Editor*] to create a custom expression. 
+. In the *Data Type* dropdown, select the data type you want to use for the variable. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, `Boolean`, `Date`, `DateTime` and `List`. If you select *List* in the Data Type field, in the *List Item Type* field, select:
+.. *String*, *Number*, or *Boolean* to return a list containing primitive data types. 
+.. *Object* to return a list of objects, for example, the output of the *Get Records* action.
+
++
+The *Label* field appears. Repeat the previous steps until you have added all the required fields. 
```

Comment:
```suggestion
Repeat the previous steps until you have added all required fields. 
```

Reply:
I am not sure what "Repeat the previous steps until you have added all the required fields" means in this context. What are you repeating?